lib/schema.js

```diff
@@ -5,14 +5,16 @@ var ajv = require('ajv')();
 var fs = P.promisifyAll(require('fs'));
 var path = require('path');
 var preq = require('preq');
+var yaml = require('js-yaml');
 
 
 /** @const */
 var SCHEMA_REPO = path.join(__dirname, '../schemas');
 
 
-function schemaPath(schema) {
-    return path.join(SCHEMA_REPO, schema.toLowerCase() + '.js');
+function schemaPath(schema, ext) {
+    ext = ext || 'js';
```

Comment:
I'd say so as well, but it seems we are using [the `js` extension](https://github.com/wikimedia/restevent/tree/8873af9125df71b2d99335b87e748a3797063aa9/schemas)